src/main/resources/hudson/plugin/versioncolumn/Messages.properties

```diff
@@ -1,3 +1,6 @@
-VersionMonitor.DisplayName=Version
+VersionMonitor.DisplayName=Agent Version
```

Comment:
"Remoting version" 🐛 . Just because agents may have different versioning (e.g. in Swarm plugin)